### PR TITLE
FUSETOOLS2-1925: Migrate VSCode Camel K extension readme file to new format

### PR DESCRIPTION
**Pull request by djelinek**
Signed-off-by: Dominik Jelinek <djelinek@redhat.com>

### CODE REVIEW

File: docs/content/logs.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+If you want to explicitly view the running log for a published integration in a new Log view, right-click on a running integration and then select the **Follow log for running Apache Camel K Integration** option.
+
+<p align="center"><img src="../images/camelk-integrations-view-remove-menu.jpg" alt="Apache Camel K Integrations view - Follow log for running integration" class="zoom" width="40%"/></p>
+
+A new editor view opens. It is named for the running Kubernetes pod where the integration is running. This log updates as new data is added:
+
+<p align="center"><img src="../images/camelk-integrations-view-integrations-log-revised.jpg" alt="Apache Camel K Integrations view - Integration log" class="zoom" width="60%"/></p>
+
+**Note**: By default, `Auto-scroll` is checked and new entries in the log will automatically be scrolled to at the bottom of the window. If turned off, the scrolling stops and can be done manually.
```
Comment (unsortedhashsets) [inline docs/content/logs.md:50]: to **at** the bottom -> to the bottom

File: docs/content/java-support.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Java Language Support
+
+From version 0.0.32 we provide [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [VS Code Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) as dependencies.
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:3]: WDYT, use the name as in the marketplace: 
- VS Code Language support for java -> Language Support for Java(TM) by Red Hat
- VS Code Debugger for Java -> Debugger for Java

File: docs/content/java-support.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+With [VS Code Language support for java](https://marketplace.visualstudio.com/items?itemName=redhat.java), you will benefit from Java Language completion on standalone files. An invisible project is created with a default classpath.
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:5]: WDYT, use the name as in the marketplace: 
- VS Code Language support for java -> Language Support for Java(TM) by Red Hat

File: docs/content/java-support.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+The command `Refresh local Java classpath for Camel K standalone file based on current editor. Available with kamel 1.4+.` refreshes specific dependencies for the classpath of the opened Integration, such as dependencies declared as part of the modeline of the file. When using a modeline to configure such dependencies, you gain a [CodeLens](https://code.visualstudio.com/blogs/2017/02/12/code-lens-roundup) link (`Refresh classpath dependencies`) at the top of the editor to trigger the refresh more easily.
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:7]: on current editor -> on **the** current editor

File: docs/content/java-support.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+Be aware of the following **limitations**:
+
+- It requires Camel K 1.4.0
+- If mistakenly called with Camel K 1.3.2-, need to restart VS Code for basic dependencies to be available again.
+- It supports modeline dependencies notation from local build. See [apache/camel-k#2213](https://github.com/apache/camel-k/issues/2213)
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:13]: from local build -> from **the** local build

File: docs/content/java-support.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+- A single classpath is provided. It means that refresh command needs to be called when switching between Integration file written in Java that does not have the same dependencies.
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:14]: that refresh command -> that **the** refresh command

File: docs/content/java-support.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+- There is no progress indicator. Please be patient. The first time may take several minutes on a slow network.
+  
+[VS Code Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) is provided to benefit from Java debug on standalone files. To leverage it, you need to start an integration, then there are 2 solutions:
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:17]: WDYT, use the name as in the marketplace:
- VS Code Debugger for Java -> Debugger for Java

File: docs/content/java-support.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Right-click on integration in Integrations view, then choose `Start Java debugger on Camel K integration`.
```
Comment (unsortedhashsets) [inline docs/content/java-support.md:19]: in Integrations view -> in **the** Integrations view

File: docs/content/settings.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
+## Apache Camel K Extension Settings
+
+To access **Tooling for Apache Camel K** extension settings:
+
+1. From the VS Code IDE, select **File > Preferences > Settings**.
+2. Select Extensions and then select **Apache Camel K Tooling Extension Settings**.
+
+<p align="center"><img src="../images/camelk-integrations-view-settings.png" alt="Apache Camel K Extension Settings" class="zoom" width="75%"/></p>
+
+Extension settings includes:
```
Comment (unsortedhashsets) [inline docs/content/settings.md:10]: settings include**s** -> settings include

File: docs/content/settings.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+- **Close Log View When Integration Removed** - Indicates whether an open Log view for a published integration should be closed if the integration is removed (stopped). Defaults to true (checked). If unchecked, the Log will remain open and indicate that the integration has been stopped in the title bar.
+- **Show Status Bar Messages** - Indicates whether to show messages in the status bar to indicate when the system is updating, such as when the Camel K Integrations view is being refreshed or a new Integration is being deployed.
+- **Namespace** - The namespace to be used for all Apache Camel K and Kubernetes operations. This defaults to `default` but can be set to whatever your namespace has been configured to in your runtime. When it is updated, the Apache Camel K Integrations view is refreshed automatically. (See [Changing the Namespace](../content/namespaces.md) below for a bit more detail.)
+- **Operator id** - Camel K can be installed with different operator id allowing multi-tenancies and mixing versions of Camel K on same cluster. The operator id can be specified in this case with this setting. If provided, all operations using `kamel` CLI underneath will use it.
```
Comment (unsortedhashsets) [inline docs/content/settings.md:15]: Camel K on same cluster -> Camel K on **the** same cluster

**Review by unsortedhashsets — COMMENTED** [review]
Hi, @djelinek 

It is really nice idea to migrate to gh-pages.
I have made some comments.

**Review by unsortedhashsets — COMMENTED** [review]
Some problems still remain, the most important of them is [broken_link_404](https://github.com/camel-tooling/vscode-camelk/pull/1402/files#diff-d1788889d65590916cf35374494cf0e75abcf59000690e82b8b04e018122be0eR6) from my point of view

**Review by mlorinc — APPROVED** [review]

**Comment by djelinek** [comment]
> Some problems still remain, the most important of them is [broken_link_404](https://github.com/camel-tooling/vscode-camelk/pull/1402/files#diff-d1788889d65590916cf35374494cf0e75abcf59000690e82b8b04e018122be0eR6) from my point of view

@unsortedhashsets yes, you were right. the examples were moved to dedicated place. I've fixed the link and also other notes were resolved

**Review by unsortedhashsets — APPROVED** [review]
LGTM

**Review by apupier — APPROVED** [review]

**Pull request merged**